doc/man3/EVP_DigestSignInit.pod

```diff
@@ -50,6 +58,11 @@ The B<EVP> interface to digital signatures should almost always be used in
 preference to the low level interfaces. This is because the code then becomes
 transparent to the algorithm used and much more flexible.
 
+EVP_DigestSign() is a single part operation which signs a single block of data
```

Comment:
"one step" is the common term I'm use to (which might just be an Americanism, too).